src/style.css

```
@@ -1,3 +1,79 @@
+html {
+  --hoverColor: #00acdc;
+}
+
 body {
   margin: 0;
+  background: #e5e5e5;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+}
+
+.header {
+  display: flex;
+  background-color: #fff;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 40px;
+  position: relative;
+  box-sizing: border-box;
+}
+
+@media screen and (width: 1200px) {
+  .header {
+    width: 1200px;
+  }
+}
+
+@media screen and (width: 1024px) {
+  .header {
+    width: 1024px;
+  }
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  text-transform: uppercase;
+  font-weight: 500;
+  padding: 0;
+  margin: 0;
+}
+
+.list__item {
+  display: inline-block;
+}
+
+.nav__link {
+  display: inline-block;
+  text-decoration: none;
```

Add position relative for correct pseudo element positioning
```suggestion
.nav__link {
  display: inline-block;
  position: relative;
  text-decoration: none;
```